_data/Sojan672/Sojan672.md

```diff
@@ -0,0 +1,5 @@
+name: SOJAN CHANDY
```

Comment:
```suggestion
name: SOJAN CHANDY
```
you forget ---
It should like this-

---
name: SOJAN CHANDY
institution: FR. CONCEICAO RODRIGUES COLLEGE OF ENGINEERING
profile_pic: Sojan672.jpg
quote: Eat-Sleep-Code-Repeat
github_user: Sojan672
---